consensus/scp/src/node.rs

```diff
@@ -32,8 +29,8 @@ pub struct Node<V: Value, ValidationError: Display> {
     /// Local node quorum set.
     pub Q: QuorumSet,
 
-    /// Map of last few slot indexes -> slots.
-    pub pending: LruCache<SlotIndex, Slot<V, ValidationError>>,
```

Comment:
Replacing this with a single Slot instance is one of the main changes of this PR. It doesn't make sense to run more than one non-externalized slot at a time. Using a cache also has some negative side effects: if a node handled a bunch of (malicious) messages for different slots, the cache could "flush" the slot it's supposed to be working on, which would break the protocol by resetting the node's state for that slot.